pkg/xds/generator/outbound_proxy_generator.go

```diff
@@ -9,6 +9,7 @@ import (
 	envoy_names "github.com/kumahq/kuma/pkg/xds/envoy/names"
 
 	kuma_mesh "github.com/kumahq/kuma/api/mesh/v1alpha1"
+	mesh_proto "github.com/kumahq/kuma/api/mesh/v1alpha1"
```

Comment:
Duplicate import. Yours is the preferred import name, so delete the old one? Although that makes the diff big, so no worries if you prefer to leave this alone :)

Reply:
I think I'll keep this consistent with what is already in the file, rm the duplicate import and change my additions to use existing import.